evm/src/generation/mod.rs

```diff
@@ -90,9 +90,16 @@ pub(crate) fn generate_traces<F: RichField + Extendable<D>, const D: usize>(
 
     let cpu_trace = trace_rows_to_poly_values(cpu_rows);
     let keccak_trace = all_stark.keccak_stark.generate_trace(keccak_inputs);
+    let keccak_memory_trace = all_stark.keccak_memory_stark.generate_trace(vec![], 0); // TODO
```

Comment:
Not sure which parameters to use here. Do we already have this data here?

Reply:
Sorry I forgot this, it should pass `operations = keccak_memory_inputs` (from `state` which is unpacked a few lines above), and `min_rows = 1 << cap_height` (looks like we don't have access to that config yet so we'd need to add it). I can add it afterward if it's easier.

Reply:
Thanks! Made the change